src/plugins/mf_priority.cpp

```diff
@@ -950,6 +954,49 @@ static int validate_cb (flux_plugin_t *p,
                                            "user has max active jobs");
     }
 
+    if (jobspec == NULL) {
+        flux_jobtap_reject_job (p,
+                                args,
+                                "job.validate: failed to unpack jobspec");
+    } else {
+        // if a queue was not unpacked, just set it to ""
+        queue_str = queue ? queue : "";
+        // count resources requested for the job
+        if (job.count_resources (jobspec) < 0) {
+            return flux_jobtap_reject_job (p,
+                                           args,
+                                           "job.validate: unable to count " \
+                                           "resources for job");
+        }
+        if (job.ncores > 0 && job.nnodes == 0)
+            // the job specified cores but no nodes, so we need to set
+            // nnodes == 1 here
+            job.nnodes = 1;
```

Comment:
Suggestion: add braces around this block even though unnecessary to enclose the code along with the multiline comment. This is also consistent with the conditionals below.

Reply:
Also, just a question: should this (adjusting nnodes to 1) b be done inside of `job.count_resources()` by default? Sorry if that's a dumb question...

Reply:
Putting it in this PR makes sense to me!